matter/efr32/efr32mg24/BRD4187C/autogen/sl_event_handler.c

```diff
@@ -31,13 +31,19 @@
 #ifdef SL_WIFI
 #include "sl_spidrv_instances.h"
 #endif
+#ifdef CHIP_917
+#include "btl_interface.h"
+#include "sl_mpu.h"
```

Comment:
I believe you don't need either of those and neither the bootloader_init().

It will result in a cleaner file with less #ifdef if you remove it

Reply:
Removed